android/app/src/main/java/org/fossasia/openevent/core/feed/facebook/api/CommentItem.kt

```diff
@@ -0,0 +1,40 @@
+package org.fossasia.openevent.core.feed.facebook.api
+
+import android.os.Parcel
+import android.os.Parcelable
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.databind.PropertyNamingStrategy
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy::class)
+data class CommentItem(
+        val createdTime: String? = null,
+        val from: Commenter? = null,
+        val message: String? = null,
+        val id: String? = null
+) : Parcelable {
```

Comment:
Use kotlin extension `@Parcelize`

Reply:
I have used it. I have also made the Commenter class implement Parcelable